explorer/src/entity/JobRun.ts

```diff
@@ -143,9 +150,15 @@ export const count = async (
 }
 
 export const saveJobRunTree = async (db: Connection, jobRun: JobRun) => {
+  const normalizeHash = (hash: string) => {
+    if (hash && hash.includes('0x')) return hash.split('0x')[1]
```

Comment:
Let's make this a private top level method so we can use it with the `searchBuilder` method.